example.py

```diff
@@ -12,9 +14,12 @@
 
 if __name__ == "__main__":
 
+    # Show information about current operation
+    logging.basicConfig(level=logging.INFO)
+
     # List of URLs to be checked
     urls = [
-        "https://XXX-iz.impfterminservice.de/impftermine/suche/XXXX-XXXX-XXXX/YYYYY/",
+        "https://XXX-iz.impfterminservice.de/impftermine/suche/XXXX-XXXX-XXXX/YYYYY/"
```

Comment:
this is a stylistic regression - just think about what happens if you duplicate the line to add another url.
or how many lines are affected by the diff if you add the missing commas.